packages/scene-graph-schema/definitions/Layout.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "id": "/definitions/Layout.json",
+
+  "definitions": {
+    "Layout": {
+      "id": "#Layout",
+      "type": "object",
+      "properties": {
+        "layoutSize": {
+          "type": "object",
+          "properties": {
+            "width":  { "type": "number" },
+            "height": { "type": "number" }
+          },
+          "required": [
+            "width",
+            "height"
+          ]
+        },
+        "resize":    { "type": "number" },
+        "layoutType": { "type": "number" },
+        "cellSize": {
+          "type": "object",
+          "properties": {
+            "width":  { "type": "number" },
+            "height": { "type": "number" }
+          },
+          "required": [
+            "width",
+            "height"
+          ]
+        },
+        "startAxis":            { "type": "number" },
+        "paddingLeft":          { "type": "number" },
+        "paddingRight":         { "type": "number" },
+        "paddingTop":           { "type": "number" },
+        "paddingBottom":        { "type": "number" },
+        "spacingX":             { "type": "number" },
+        "spacingY":             { "type": "number" },
+        "verticalDirection":    { "type": "number" },
+        "horizontalDirection":  { "type": "number" }
+      }
```

Comment:
Is there any required property in Layout root ?

Reply:
Added because `layoutType` and `resize` were required.
https://github.com/drecom/scene-graph/pull/27/commits/520fa7891958c63365c3d2aaf40929e473e83ccf